docs/topics/Gateway.md

```diff
@@ -684,6 +688,47 @@ Sent when a guild role is deleted.
 | guild_id | snowflake | id of the guild |
 | role_id  | snowflake | id of the role  |
 
+### Invites
+
+### Invite Create
+
+Sent when a new invite to a channel is created.
+
+###### Invite Create Event Fields
+
+| Field      | Type                                                                          | Description                                                                                                        |
+| ---------- | ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| channel_id | snowflake                                                                     | the channel the invite is for                                                                                      |
+| code       | string                                                                        | the unique invite [code](#DOCS_RESOURCES_INVITE/invite-object)                                                     |
+| created_at | timestamp                                                                     | the time at which the invite was created                                                                           |
+| guild_id   | snowflake                                                                     | the guild of the invite                                                                                            |
+| inviter    | [partial user object](#DOCS_TOPICS_GATEWAY/invite-create-partial-user-object) | the user that created the invite                                                                                   |
```

Comment:
we should just clone https://discordapp.com/developers/docs/resources/invite#invite-metadata-object behavior and link to the user object as this event is just serializing the invite the same way. then we avoid redocumenting user

Reply:
I notice here that we actually serialize this different than our normal invite object. Our invite object from our REST API returns `guild` and `channel` objects in which are `id` fields, but this event flattens them out. From our API:

```json
"code": "123456",
    "guild": {
        "id": "290926798626999999",
        "name": "Mason's Test Server",
        "splash": "75610b05a0dd09ec2c3c7df9f6975ea0",
        "banner": "5c3cb8d1bc159937fffe7e641ec96ca7",
        "description": null,
        "icon": "389030ec9db118cb5b85a732333b7c98",
        "features": [
            "BANNER",
            "NEWS",
            "INVITE_SPLASH",
            "VANITY_URL",
            "COMMERCE",
            "VERIFIED",
            "VIP_REGIONS"
        ],
        "verification_level": 0,
        "vanity_url_code": "absh12385f"
    },
    "channel": {
        "id": "651144899865089999",
        "name": "news",
        "type": 0
    },
    "inviter": {
        "id": "53908232506183680",
        "username": "Mason",
        "avatar": "a_d5efa99b3eeaa7dd43acca82f5692432",
        "discriminator": "1337"
    }
}
```

Should we normalize these?